### ISSUE TITLE
Question about pretraining and segmentation evaluation.

### ISSUE THREAD

**Issue by volgachen** [baaivision/Painter#6]
Nice work! I have some questions about the details. Hope you can help.

1. Is the model trained from scratch or pretrained?
2. As the model has a fixed input size 448*448. How to perform segmentation evaluation on images with various aspect ratios? Pad it with constant or perform sliding inference?
3. Does the training loss mask ignored pixels out, as well as padded pixels?

**Comment by WXinlong** [comment]
@volgachen  Hi,  
1. Painter uses MAE pre-trained model as initialization, please refer to [README].(https://github.com/baaivision/Painter/tree/main/Painter#training)
2. We didn't use test-time aug such as sliding inference or multi-scale inference. Instead, we directly resize the image into 448*448. The output is resized to the original size for evaluation.
3. Yes. Please refer to [models_painter](https://github.com/baaivision/Painter/blob/fd513a632b85ab4278b8f18389d30f419afedc1c/Painter/models_painter.py#L433) and [pairdataset.py](https://github.com/baaivision/Painter/blob/fd513a632b85ab4278b8f18389d30f419afedc1c/Painter/data/pairdataset.py#L190) for details.
